ExtendedControls/Controls/AstroPlot/Matrix.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+
+namespace ExtendedControls.Controls.AstroPlot
+{
```

Comment:
So, there are already Matrix in say System.Drawing.Drawing2D.. As this is a 2d matrix, then it should be called Matrix2.  Also, this does not belong in astroplot, if we are going to make a 2d matrix class, it should be part of baseutils.

Reply:
Good point. Actually, they are matrix transformation metjods, so I could call them differently. Anyway, we can move them to Baseutils, yep...